### ISSUE TITLE
Redundant prereqs Test2::*

### ISSUE THREAD

**Issue by pghmcfc** [atoomic/XString#2]
The dist.ini has a bunch of additional prereqs that are not actually used in the dist:
```
Test2::Bundle::Extended 	= 0
Test2::Tools::Explain 		= 0
Test2::Plugin::NoWarnings 	= 0
```
Could these be removed?

**Comment by toddr** [comment]
Strange. This should have closed.

**Comment by toddr** [comment]
Fixed by #8
